Makefile

```diff
@@ -15,7 +15,7 @@ test:
 # ---- Documentation ---
 
 update-docs:
-	pip3 install sphinx==3.5.4 sphinxcontrib.httpdomain sphinx-rtd-theme  # sphinx4 is not supported yet by sphinx-contrib/httpdomain, see https://github.com/sphinx-contrib/httpdomain/issues/46
```

Comment:
I was unable to run `make update-docs` without putting back these 2 dependencies. Maybe you happened to have them installed already?

Reply:
I believe you are right and these two need to be put back.